app/services/forest_liana/schema_utils.rb

```diff
@@ -2,9 +2,14 @@ module ForestLiana
   class SchemaUtils
 
     def self.associations(active_record_class)
-      active_record_class
-        .reflect_on_all_associations
-        .select { |association| !polymorphic?(association) && !is_active_type?(association.klass) }
+      associations = active_record_class.reflect_on_all_associations.select do |association|
+        next if association.options[:polymorphic] && !is_active_type?(association.klass)
```

Comment:
I am not sure you should apply directly the code snippet suggested in the issue https://github.com/ForestAdmin/forest-rails/issues/420.

`next` does not look to be the most readable way to go, it may work to return `next (nil)`, but `return false` is more explicit in `select` block IMO.

I don't get why `!is_active_type?(association.klass)` is not in the `begin`/`rescue` as it is the `association.klass` that looks to be problematic in some cases.